RestApi/Sepes.RestApi/Startup.cs

```diff
@@ -79,7 +79,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
             app.UseCors(MyAllowSpecificOrigins);
-            app.UseHttpsRedirection();
+
+            // UseHttpsRedirection doesn't work well with docker.
+            if(this._config.httpOnly) {
```

Comment:
The logic may be flawed. This only uses HTTPS redirection if the configuration is set to use HTTP